flexmeasures/data/schemas/scheduling/storage.py

```diff
@@ -54,36 +54,62 @@ class StorageFlexModelSchema(Schema):
     You can use StorageScheduler.deserialize_flex_config to get that filled in.
     """
 
-    soc_at_start = fields.Float(required=True, data_key="soc-at-start")
+    soc_at_start = QuantityField(
+        required=True,
+        to_unit="MWh",
+        default_src_unit="dimensionless",  # placeholder, overridden in __init__
+        return_magnitude=True,
+        data_key="soc-at-start",
+    )
 
-    soc_min = fields.Float(validate=validate.Range(min=0), data_key="soc-min")
-    soc_max = fields.Float(data_key="soc-max")
+    soc_min = QuantityField(
```

Comment:
No, because these can also be set as a sensor attribute or as an asset attribute (see `data.models.planning.storage.MetaStorageScheduler.ensure_soc_min_max`).